addons/binding/org.openhab.binding.network/README.md

```diff
@@ -117,9 +117,13 @@ Linux has three different tools:
 *   arping of the ip-utils (Ubuntu/Debian: `apt-get install iputils-arping`)
 *   arping by Thomas Habets (Ubuntu/Debian: `apt-get install arping`)
 
-arping by Thomas Habets runs on Windows and MacOS as well.
+arping by Thomas Habets runs on Windows and MacOS as well. However versions older than 2.18 do not support an advantageous parameter combination (-C 1 -w <timeout>), thus making iputils-arping the preffered solution for raspbian/openHABian.
```

Comment:
... for Linux based system. (No bias please. There's also fedora etc)

Reply:
I'm not sure this text is accurate, older versions of TH's `arping` support the parameters, but they have different time units and capitalization. For the 2.14 version, the -w parameter is microseconds, and the -C is lower case instead.